### ISSUE TITLE
Making Ankisync compatible with the new Anki database

### ISSUE THREAD

**Issue by patarapolw** [patarapolw/ankisync2#3]
See [Anki database structure has changed](https://github.com/patarapolw/ankisync/issues/10)

**Comment by patarapolw** [comment]
Does anyone know how to get over `unicase collation`?

```
sqlite3.OperationalError: no such collation sequence: unicase
```

It seems to be based on Rust SQLite.

https://github.com/ankitects/anki/commit/4ff17d31b3d35ca39944a0f2829e457e6b5dcdda

**Comment by patarapolw** [comment]
Solved with `.where(Model.col_name.collate("NOCASE") == "xxx")` (or `"BINARY"`)

**Comment by AlexRiina** [comment]
I'm not familiar with this project, but just ran into the same issue on my own project and found this issue. I solved the issue by registering a new collation named unicase after opening my sqlite connection. Unicase is a complex beast that replaces out unicode characters so that strings like Maße and MASSE are considered equal in the database. 

I'm not sure I've reproduced it exactly, but I ended up with (**updated**)

```python
from unidecode import unidecode

def unicase_compare(x, y):
    x_ = unidecode(x).lower()
    y_ = unidecode(y).lower()
    return 1 if x_ > y_ else -1 if x_ < y_ else 0

connection.create_collation("unicase", unicase_compare)
```

here's the rust library for the actual unicase collation https://github.com/seanmonstar/unicase
